### PR TITLE
WL-4003 Adding mutiple files to a Lessons page from Resources tool.

### PR DESCRIPTION
**Pull request by ouit0408**
Deleted the previous pull request and this one has the fix for the problem of redirection after adding multiple Resources from 'Add Content Link'

### CODE REVIEW

File: tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java [hunk 1]
```diff
@@ -1142,73 +1142,79 @@ public String processResource(int type, boolean isWebSite, boolean isCaption) {
...
+				pushed = pushAdvisor();
+				ContentResourceEdit res = contentHostingService.editResource(id);
+				res.setContentType("text/vtt");
+				contentHostingService.commitResource(res, NotificationService.NOTI_NONE);
+			} catch (Exception ignore) {
+				return "no-reference";
+			}finally {
+				if(pushed) popAdvisor();
+			}
+		}boolean pushed = false;
```
Comment (buckett) [inline tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java:1215]: Bad formatting.

File: tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java [hunk 2]
```diff
@@ -1142,73 +1142,79 @@ public String processResource(int type, boolean isWebSite, boolean isCaption) {
...
 		} else {
 			return "cancel";
 		}
 
-		boolean pushed = false;
+		return returnMesssage;
+	}
+
+	//This method is written to enable user to select multiple Resource from the tool as per Jira WL-4003
+	private String processMultipleResource(Reference reference,int type, boolean isWebSite, boolean isCaption){
```
Comment (buckett) [inline tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java]: `processSingleResource()` makes more send to me as it's caller is the one processing multiple resources.

File: tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java [hunk 3]
```diff
@@ -1142,73 +1142,79 @@ public String processResource(int type, boolean isWebSite, boolean isCaption) {
...
+
 		} else {
 			return "cancel";
 		}
 
-		boolean pushed = false;
+		return returnMesssage;
+	}
+
+	//This method is written to enable user to select multiple Resource from the tool as per Jira WL-4003
```
Comment (buckett) [inline tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java]: Local jira reference.

**Comment by ouit0408** [comment]
Made changes and pushed the code.

**Pull request merged**